catalog-rest-service/src/main/resources/json/schema/metadataIngestion/databaseServiceMetadataPipeline.json

```diff
@@ -109,6 +109,10 @@
       "description": "Regex exclude tables or databases that matches the pattern.",
       "$ref": "../type/filterPattern.json#/definitions/filterPattern"
     },
+    "databaseFilterPattern": {
```

Comment:
This will need to be eventually added to the UI cc @darth-coder00 @vivekratnavel

Reply:
lets add this in this release itself